<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SecDialog</class>
 <widget class="QDialog" name="SecDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>829</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_presentation">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>751</width>
     <height>101</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Vous pouvez visualiser certaines grandeurs physiques mesurées par votre station météo pour une date donnée.</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>230</y>
     <width>261</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_temperature">
      <property name="text">
       <string>Température</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_humidite">
      <property name="text">
       <string>Humidité</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_vitesse_du_vent">
      <property name="text">
       <string>Vitesse du vent</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_direction_du_vent">
      <property name="text">
       <string>Direction du vent </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_quitter">
      <property name="text">
       <string>Quitter</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>321</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_date">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>Entrez la date :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_saisie_date">
      <property name="contextMenuPolicy">
       <enum>Qt::DefaultContextMenu</enum>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string>2018-06-01</string>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
      <property name="cursorMoveStyle">
       <enum>Qt::LogicalMoveStyle</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_images">
   <property name="geometry">
    <rect>
     <x>34</x>
     <y>225</y>
     <width>441</width>
     <height>331</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_affichage">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>140</y>
     <width>251</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_quitter</sender>
   <signal>clicked()</signal>
   <receiver>SecDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>623</x>
     <y>471</y>
    </hint>
    <hint type="destinationlabel">
     <x>650</x>
     <y>513</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
